src/Containers.scss

```diff
@@ -4,6 +4,17 @@
         padding-top: var(--pf-global--spacer--md);
     }
 
+    .pod-header-details {
+	border-color: #ddd;
+	margin-top: var(--pf-global--spacer--md);
+	margin-left: var(--pf-global--spacer--md);
+	margin-right: var(--pf-global--spacer--md);
+    }
```

Comment:
stylelint did not complain about this? :)

Reply:
We don't have stylelint for cockpit-podman yet, it's on my list to add after adding it to starterkit.